src/shader.wgsl

```diff
@@ -70,11 +79,11 @@ fn vs_main(in_vert: VertexInput) -> VertexOutput {
     vert_output.position.y *= -1.0;
 
     vert_output.color = vec4<f32>(
-        f32((color & 0x00ff0000u) >> 16u),
-        f32((color & 0x0000ff00u) >> 8u),
-        f32(color & 0x000000ffu),
-        f32((color & 0xff000000u) >> 24u),
-    ) / 255.0;
+        srgb_to_linear((color & 0x00ff0000u) >> 16u),
+        srgb_to_linear((color & 0x0000ff00u) >> 8u),
+        srgb_to_linear(color & 0x000000ffu),
```

Comment:
This conversion should happen only if the `ColorMode` chosen is `Accurate`. The legacy `Web` mode breaks here, since blending should happen directly in the sRGB color space.

It may be better to conditionally perform the conversion on the CPU before upload, if possible.

`Accurate`:
![image](https://github.com/grovesNL/glyphon/assets/518289/27991207-82aa-484e-9914-65da0bc1206d)

`Web`:
![image](https://github.com/grovesNL/glyphon/assets/518289/ac96bfb8-7113-4a86-bfd3-ffaf54ee5df8)

Reply:
Makes sense, thanks for checking this! I'll see if we can change the upload path instead